src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRowForFeedbackParticipant.tag

```diff
@@ -0,0 +1,76 @@
+<%@ tag trimDirectiveWhitespaces="true" %>
+<%@ tag description="Feedback Response Comment" pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ taglib tagdir="/WEB-INF/tags/shared" prefix="shared" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="frc" type="teammates.ui.template.FeedbackResponseCommentRow" required="true" %>
+<%@ attribute name="responseIndex" %>
+<%@ attribute name="qnIndex" %>
+<%@ attribute name="isSessionOpenForSubmission" type="java.lang.Boolean"%>
+<%@ attribute name="moderatedPersonEmail" %>
+<%@ attribute name="submittable" %>
+<%@ attribute name="isPreview" %>
+<%@ attribute name="isModeration" %>
+
+<c:set var="divId" value="${qnIndex}-${responseIndex}" />
+<c:set var="deleteLink"><%= Const.ActionURIs.FEEDBACK_PARTICIPANT_FEEDBACK_RESPONSE_COMMENT_DELETE %></c:set>
+
+<li class="list-group-item list-group-item-warning" id="responseCommentRow-${divId}">
+  <div id="commentBar-${divId}" class="row">
+    <div class="col-xs-10">
+    <span class="text-muted">
+          Your comment about the feedback.
```

Comment:
If we have a link that describes this is a box to add comment, we may remove this.
In doing so, you can also use this tag in the instructor's result page.

Reply:
This is for the scenario when comment has already been added.
We are using `Your comment about the above response` when we show add form. 
Should they be same?
> If we have a link that describes this is a box to add comment, we may remove this.

I think removing it makes the form look very plain. :sweat_smile:

Reply:
Just a suggestions for you, because you will use this tag in the instructor side and displaying of `You comment about the feedback` shouldn't be there.

Reply:
But you said on slack, to not include feedback participant comments in instructor modal?

Reply:
And If we plan to include feedback participant comments in modal (both options work for me), this is how it'll look currently.
![modal](https://user-images.githubusercontent.com/24381442/43216603-0d727394-905d-11e8-81ab-7cbde22f7bf6.png)

I feel it looks quite odd. What do you think

Reply:
> What do you think

I think a bit weird to. So we shall not include it in the PR

> But you said on slack, to not include feedback participant comments in instructor modal?

Yes. Not in the add comment instructor modal but should display it on GRQ or RGQ view in the instructor side.

Reply:
@xpdavid Sorry, seems I missed this view.

Okay so I tried both the UIs.

![viewss](https://user-images.githubusercontent.com/24381442/43254724-9f5e4bc4-90e5-11e8-8695-32d872633eaa.png)

![withoutheader](https://user-images.githubusercontent.com/24381442/43254739-a7b6f8de-90e5-11e8-999b-5fff452bbbe6.png)

Which one looks better?

Reply:
The first one I think.